src/compas_robots/resources/basic.py

```diff
@@ -165,7 +165,10 @@ def __init__(self, path, support_package):
         super(LocalPackageMeshLoader, self).__init__()
         self.path = path
         self.support_package = support_package
-        self.schema_prefix = "package://" + self.support_package + "/"
+        if support_package is None or support_package == "":
+            self.schema_prefix = "package://"
```

Comment:
Please add a note in the docstring indicating `support_package` is optional, and add a default `None` to the param itself.